src/table/settings/network_secrets_settings.ini

```diff
@@ -61,6 +61,21 @@ flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC | SF_NETWORK_ONLY
 def      = nullptr
 cat      = SC_BASIC
 
+[SDTC_SSTR]
+var      = network.client_secret_key
+type     = SLE_STR
+length   = NETWORK_SECRET_KEY_LENGTH
+flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC
+def      = nullptr
+
+[SDTC_SSTR]
+var      = network.client_public_key
+type     = SLE_STR
+length   = NETWORK_SECRET_KEY_LENGTH
+flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC
+def      = nullptr
+pre_cb   = [](auto) { return false; }
```

Comment:
What is the effect of this `pre_cb`?

Reply:
It prevents that you change your public key from the console, as doing that is essentially pointless and might cause further harm down the line when the client and server are in disagreement about the public key of the client.

Reply:
Maybe add that as a comment here? As I am sure the next person looking at it would have the same question :)